Update LinkedIn Audiences (#7220)
segmentio/segment-docs: src/connections/destinations/catalog/actions-linkedin-audiences/index.md

{% include components/actions-fields.html settings="true"%}

> ## Note for Segment Linked Audiences

Reviewer: ```suggestion
## Linked Audiences
```

> For Linked Audiences, you must set "Add" or "Remove" as the value for the **DMP User Action** field, as the **Auto Detect** option is not supported.

Reviewer: ```suggestion
If you're using Linked Audiences, you must set *Add* or *Remove* as the value for the **DMP User Action** field, as Linked Audiences doesn't support the **Auto Detect** option.
```

## Troubleshooting

### Error: Action :: field is required but not found

> This error occurs when the **DMP User Action** field is not set to "Add" or "Remove" in the mapping. Or it's set to "Auto Detect," which comes with a couple of caveats:

Reviewer: ```suggestion
This error occurs when the **DMP User Action** field isn't set to *Add* or *Remove* in the mapping, or it's set to *Auto Detect,* which comes with a couple of caveats:
```

> - For Linked Audiences, you must set "Add" or "Remove" as the value for the **DMP User Action** field, as the **Auto Detect** option is not supported.

Reviewer: ```suggestion
- For Linked Audiences, you must set *Add* or *Remove* as the value for the **DMP User Action** field, as the **Auto Detect** option isn't supported.
```

> - For Profile Audiences, it must match the default "Enter Event" names set in the destination settings at the Audience level. Those defaults are "Audience Entered" and "Audience Exited."

Reviewer: ```suggestion
- For Profile Audiences, it must match the default *Enter Event* names set in the destination settings at the Audience level. Those defaults are *Audience Entered* and *Audience Exited*.
```
